typescript/infra/config/aggregationIsm.ts

```diff
@@ -1,103 +1,28 @@
+// Aggregation (1/2)
```

Comment:
think this moved unintentionally

and actually, I removed this file in favor of the new `routingIsm`